site/content/docs/cicd/settings.md

```diff
@@ -0,0 +1,35 @@
+# Deployment Settings File
```

Comment:
This content is a duplicate of what is in cicd.md, can we please delete this file?